### ISSUE TITLE
Issues running Waterfall + Geyser + Floodgate

### ISSUE THREAD

**Issue by Jafesu** [GeyserMC/pterodactyl-stuff#45]
When the egg runs i get the following error.

```
[01:07:59 WARN]: Could not load plugin from file plugins/floodgate-bungee.jar
java.util.zip.ZipException: zip END header not found
        at java.util.zip.ZipFile$Source.findEND(ZipFile.java:1469) ~[?:?]
        at java.util.zip.ZipFile$Source.initCEN(ZipFile.java:1477) ~[?:?]
        at java.util.zip.ZipFile$Source.<init>(ZipFile.java:1315) ~[?:?]
        at java.util.zip.ZipFile$Source.get(ZipFile.java:1277) ~[?:?]
        at java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:709) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:243) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:172) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:347) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:318) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:284) ~[?:?]
        at net.md_5.bungee.api.plugin.PluginManager.detectPlugins(PluginManager.java:427) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
        at net.md_5.bungee.BungeeCord.start(BungeeCord.java:276) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
        at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:67) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
        at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
[01:07:59 WARN]: Could not load plugin from file plugins/Geyser-BungeeCord.jar
java.util.zip.ZipException: zip END header not found
        at java.util.zip.ZipFile$Source.findEND(ZipFile.java:1469) ~[?:?]
        at java.util.zip.ZipFile$Source.initCEN(ZipFile.java:1477) ~[?:?]
        at java.util.zip.ZipFile$Source.<init>(ZipFile.java:1315) ~[?:?]
        at java.util.zip.ZipFile$Source.get(ZipFile.java:1277) ~[?:?]
        at java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:709) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:243) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:172) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:347) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:318) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:284) ~[?:?]
        at net.md_5.bungee.api.plugin.PluginManager.detectPlugins(PluginManager.java:427) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
        at net.md_5.bungee.BungeeCord.start(BungeeCord.java:276) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
        at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:67) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
        at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15) ~[waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:5c1e956:504]
```

When i look at the jar files, all it shows is

```
<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx/1.14.2</center>
</body>
</html>
```

**Comment by NakkiiBrks** [comment]
Same problem here, but with paper+floodgate egg

```
[17:35:59 ERROR]: Could not load 'plugins/floodgate-spigot.jar' in folder 'plugins'
org.bukkit.plugin.InvalidDescriptionException: Invalid plugin.yml
        at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:186) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:155) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.loadPlugins(CraftServer.java:423) ~[paper-1.19.2.jar:git-Paper-211]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:278) ~[paper-1.19.2.jar:git-Paper-211]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1100) ~[paper-1.19.2.jar:git-Paper-211]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305) ~[paper-1.19.2.jar:git-Paper-211]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.util.zip.ZipException: zip END header not found
        at java.util.zip.ZipFile$Source.findEND(ZipFile.java:1469) ~[?:?]
        at java.util.zip.ZipFile$Source.initCEN(ZipFile.java:1477) ~[?:?]
        at java.util.zip.ZipFile$Source.<init>(ZipFile.java:1315) ~[?:?]
        at java.util.zip.ZipFile$Source.get(ZipFile.java:1277) ~[?:?]
        at java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:709) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:243) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:172) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:347) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:318) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:284) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:174) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        ... 6 more
[17:35:59 ERROR]: Could not load 'plugins/Geyser-Spigot.jar' in folder 'plugins'
org.bukkit.plugin.InvalidDescriptionException: Invalid plugin.yml
        at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:186) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:155) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.loadPlugins(CraftServer.java:423) ~[paper-1.19.2.jar:git-Paper-211]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:278) ~[paper-1.19.2.jar:git-Paper-211]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1100) ~[paper-1.19.2.jar:git-Paper-211]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305) ~[paper-1.19.2.jar:git-Paper-211]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.util.zip.ZipException: zip END header not found
        at java.util.zip.ZipFile$Source.findEND(ZipFile.java:1469) ~[?:?]
        at java.util.zip.ZipFile$Source.initCEN(ZipFile.java:1477) ~[?:?]
        at java.util.zip.ZipFile$Source.<init>(ZipFile.java:1315) ~[?:?]
        at java.util.zip.ZipFile$Source.get(ZipFile.java:1277) ~[?:?]
        at java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:709) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:243) ~[?:?]
        at java.util.zip.ZipFile.<init>(ZipFile.java:172) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:347) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:318) ~[?:?]
        at java.util.jar.JarFile.<init>(JarFile.java:284) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:174) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
        ... 6 more
```

**Comment by tklostermannNSD** [comment]
Someone found a solution?

**Comment by NakkiiBrks** [comment]
I just updated to an newer version and the problem was solved

**Comment by PseudoResonance** [comment]
This should be fixed in the latest commit, fbda9f5.

**Comment by onebeastchris** [comment]
With the latest commit, all eggs should be operational - therefore closing this.
